src/shared/lib/resources/proposal/sprint-with-us.ts

```diff
@@ -257,12 +257,13 @@ export interface DeleteValidationErrors extends BodyWithErrors {
   status?: string[];
 }
 
-export function isSWUProposalStatusVisibleToGovernment(s: SWUProposalStatus): boolean {
+export function isSWUProposalStatusVisibleToGovernment(s: SWUProposalStatus, role: UserType): boolean {
```

Comment:
Last minor change, please update the `UserType` type signature to a union of `UserType.Government | UserType.Admin` to exclude vendors.

Reply:
Same for CWU